README.md

```diff
@@ -8,11 +8,14 @@ See [action.yml](action.yml).
 
 ## Standard Usage
 
-Configure a workflow to run a job on these `pull_request` events:
+Configure a workflow to run a job on these `pull_request` or  `pull_request_target` events:
+
+If you enable `comments` feature we recommend to listen `pull_request_target` event. 
+`pull_request_target` event has write permission to the target repository allowing external forks create comments.
 
 ```yaml
 on:
-  pull_request:
+  pull_request_target:
     types: [opened, edited, reopened, synchronize]
```

Comment:
does pull_request_target have all the same types as pull_request?

Reply:
Yes we have the same types [reference](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target)